includes/Ajax_Handlers/WC_Stripe_Checkout_Sessions_Ajax_Handler.php

```
@@ -26,35 +28,31 @@ public function create_checkout_session(): void {
 			$wc_customer = WC()->customer;
 			if ( ! $wc_customer ) {
-				throw new Exception( __( 'Unable to retrieve customer data.', 'woocommerce-gateway-stripe' ) );
+				throw new \Exception( __( 'Unable to retrieve customer data.', 'woocommerce-gateway-stripe' ) );
 			}
 
 			$user_id = $wc_customer->get_id();
 
 			// TODO: Test guest checkout flow.
 			try {
-				$stripe_customer = new WC_Stripe_Customer( $user_id );
+				$stripe_customer = new \WC_Stripe_Customer( $user_id );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '43', column '28'). (undefined)

(MissingImport)